test/pgo_basic_SUITE.erl

```diff
@@ -204,3 +204,23 @@ types(_Config) ->
     #{command := select, rows := [Row17, Row18]} = R3,
     ?assertMatch({?TXT_UUID}, Row17),
     ?assertMatch({?TXT_UUID}, Row18).
+
+numerics(_Config) ->
+    % TODO:
+    % - Test non-param based query (i.e., select 1234:numeric(0, 1) )
+    % - Test precision and scale query params
```

Comment:
General note, please replace all single `%` (when it is the first character on a line like this) with `%%`

Reply:
k